Production/govuk_ios/Coordinators/SettingsCoordinator.swift

```diff
@@ -16,24 +17,30 @@ class SettingsCoordinator: TabItemCoordinator {
          analyticsService: AnalyticsServiceInterface,
          coordinatorBuilder: CoordinatorBuilder,
          deviceInformationProvider: DeviceInformationProviderInterface,
+         authenticationService: AuthenticationServiceInterface,
          notificationService: NotificationServiceInterface) {
         self.viewControllerBuilder = viewControllerBuilder
         self.deeplinkStore = deeplinkStore
         self.analyticsService = analyticsService
         self.coordinatorBuilder = coordinatorBuilder
+        self.authenticationService = authenticationService
         self.notificationService = notificationService
         self.settingsViewModel = SettingsViewModel(
             analyticsService: analyticsService,
             urlOpener: UIApplication.shared,
             versionProvider: Bundle.main,
             deviceInformationProvider: deviceInformationProvider,
+            authenticationService: authenticationService,
             notificationService: notificationService,
             notificationCenter: .default
         )
         super.init(navigationController: navigationController)
         self.settingsViewModel.notificationsAction = { [weak self] in
             self?.startNotificationsSettings()
         }
+        self.settingsViewModel.signoutAction = { [weak self] in
+            self?.startSignOut()
+        }
```

Comment:
Could probably move these (notificationsAction + signoutAction) to a private func which would remove the Sonar `self` warning + keep the initialiser cleaner